frontend/src/features/documents/list/DocumentListView.test.tsx

```diff
@@ -38,6 +50,13 @@ describe('Document List View', () => {
     mockAxios.reset();
     mockAxios.onGet(`documents/document-types`).reply(200, mockDocumentTypesResponse());
   });
+  afterEach(() => {
+    mockAxios.reset();
```

Comment:
Do we need to reset mockAxios before AND after each call? I'd think only one time would suffice...

Reply:
not sure, but given we are seeing some issues with concurrency I figured it would not hurt to be add the extra reset